src/other/metadata.jl

```diff
@@ -0,0 +1,117 @@
+# Defining behavior for DataFrames metadata
+abstract type AbstractMetaData end
+
+mutable struct MetaData <: AbstractMetaData
+    columndata::Dict{Symbol, Vector}
+end
+    
+MetaData() = MetaData(Dict{Symbol,Vector}())
+
+Base.isequal(x::MetaData, y::MetaData) = isequal(x.columndata, y.columndata)
```

Comment:
Do we really need these definitions?

Reply:
I thought they were standard boilerplate for new structs. they are deleted now.